### ISSUE TITLE
git-remote-https.ext crashes on blank Windows 10

### ISSUE THREAD

**Issue by ArminVieweg** [git-for-windows/git#1474]
Hi, I've recently reset my Windows 10 machine. 
I've installed git using scoop. And everytime I perform a "git fetch" I get an error box and this entry in system log:

```
Name der fehlerhaften Anwendung: git-remote-https.exe, Version: 2.16.1.2, Zeitstempel: 0x5a7491f4
Name des fehlerhaften Moduls: git-remote-https.exe, Version: 2.16.1.2, Zeitstempel: 0x5a7491f4
Ausnahmecode: 0xc0000005
Fehleroffset: 0x00000000000056cd
ID des fehlerhaften Prozesses: 0x2af4
Startzeit der fehlerhaften Anwendung: 0x01d39d28f3c5c910
Pfad der fehlerhaften Anwendung: C:\Users\armin\scoop\apps\git\current\mingw64\libexec\git-core\git-remote-https.exe
Pfad des fehlerhaften Moduls: C:\Users\armin\scoop\apps\git\current\mingw64\libexec\git-core\git-remote-https.exe
Berichtskennung: bf58423d-768b-43aa-b384-10b402573d80
Vollständiger Name des fehlerhaften Pakets: 
Anwendungs-ID, die relativ zum fehlerhaften Paket ist: 
```

I'm using version **2.16.1.windows.2** on a brand new Windows 10 set up. No clue how to fix this. Thanks for any hints!

**Comment by ArminVieweg** [comment]
Ok. When I install it with the installer from git-scm.com, it works. Nevermind.

**Comment by dscho** [comment]
I *think* that the symptom relates to an unset `http.sslBackend` (although I cannot really see how that would happen when installing Git for Windows via the installer).

**Comment by dscho** [comment]
FWIW even if it does not look like it, #1475 is probably describing this very same issue.

**Comment by dscho** [comment]
Oops. I meant #1478.

**Comment by ArminVieweg** [comment]
Should I reopen this one again? But I swear, with the installer it works ;)

**Comment by dscho** [comment]
> Should I reopen this one again?

No, as it has been fixed in v2.16.1(3) (released yesterday).

> But I swear, with the installer it works ;)

Yes, and the reason is that the installer always sets `http.sslBackend`... ;-) So the previously unguarded `strcmp("schannel", http_ssl_backend)` tried to access `NULL`, which will always fail.
